shumai/tensor/tensor.ts

```diff
@@ -294,89 +299,104 @@ export class Tensor {
       this.stats = obj.stats
       this.grad = obj.grad
       this.op = obj.op
+      if (is_tidy_run) this.track_tidy_ptr()
```

Comment:
I think we can unify the `is_tidy_run` with `creationTracker`

what about something like

```javascript
if (_tidyTracker) _tidyTracker.set(this.ptr, this)
```